trl/commands/cli_utils.py

```diff
@@ -111,7 +111,6 @@ class DPOScriptArguments:
     dataset_name: str = field(default=None, metadata={"help": "the dataset name"})
     dataset_train_split: str = field(default="train", metadata={"help": "The dataset split to use for training"})
     dataset_test_split: str = field(default="test", metadata={"help": "The dataset split to use for evaluation"})
-    sanity_check: bool = field(default=False, metadata={"help": "only train on 1000 samples"})
```

Comment:
This type of debugging arg shouldn't live in the lib IMO

Reply:
Let's remove them all